.github/workflows/ci.yaml

```diff
@@ -2190,6 +2190,16 @@ jobs:
           VIA_MAINTENANCE_MODE: "true"
         run: |
           sudo -E make e2e-qemu
+      - name: e2e-min-requirements
+        env:
+          GITHUB_STEP_NAME: ${{ github.job}}-e2e-min-requirements
+          IMAGE_REGISTRY: registry.dev.siderolabs.io
+          QEMU_MEMORY_CONTROLPLANES: "2048"
+          QEMU_MEMORY_WORKERS: "1024"
+          QEMU_SYSTEM_DISK_SIZE: "10240"
```

Comment:
won't default be okay here?

Reply:
default is 15 GiB, and we say 10 GiB is enough in the docs